Gemfile

```diff
@@ -5,7 +5,7 @@ ruby '2.7.4'
 
 # Load env variables
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 6.0.3', '>= 6.0.4.1'
+gem 'rails', '~> 6.0.4', '>= 6.0.4.2'
```

Comment:
Maybe we should relax and bump this anyway, to `~> 6.1`?

Reply:
ok!